source/getting-started/components/elasticsearch.rst

```diff
@@ -0,0 +1,42 @@
+| **wazuh** |--| **alerts**       | Stores alerts generated by the :ref:`Wazuh server <wazuh_server>`. These are created each time an event trips a rule with a high enough priority (this threshold is configurable).                                             |
++---------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+ 
+| **wazuh** |--| **archives**     | Stores all events (archive data) received by the analysis engine, whether or not they trip a rule.                                                                                                                             |
++---------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+ 
+| **wazuh** |--| **monitoring**   | Stores data related to the :ref:`Wazuh agents <wazuh_agent>` status over time. It is used by the web interface to represent when individual agents are or have been ``Active``, ``Disconnected``, or ``Never connected``.      |
++---------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+ 
+| **wazuh** |--| **statistics**   | Stores data related to the Wazuh manager performance. It is used by the web interface to represent the daemons' statistics.                                                                                                    |
++---------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+ 
+
+
+Elasticsearch is well suited for time-sensitive use cases like security analytics and infrastructure monitoring as it's a near real-time search platform. The latency from the time a document is indexed until it becomes searchable is very short — typically one second.
+
+You interact with the Elasticsearch clusters using the REST API, which offers a lot of flexibility. You can perform searches, add or delete documents, modify indices, and more. Elasticsearch comes with a wide set of features. In addition to its speed, scalability, and resiliency, Elasticsearch has several powerful built-in features that make storing and searching data even more efficient, such as data rollups, alerting, anomaly detection, and index lifecycle management.
```

Comment:
The content of the deleted sentence is repeated in the sentence right next to it. I'll delete it to avoid redundancy.

```suggestion
You interact with the Elasticsearch clusters using the REST API, which offers a lot of flexibility. You can perform searches, add or delete documents, modify indices, and more. In addition to its speed, scalability, and resiliency, Elasticsearch has several powerful built-in features that make storing and searching data even more efficient, such as data rollups, alerting, anomaly detection, and index lifecycle management.
```